index.html

```diff
@@ -21,26 +21,97 @@ <h2 class="notoc">Contents</h2>
 
 
 
+<section id="when_should_review_be_requested">
+  <h2>When should wide review be requested?</h2>
+  <p>A document is available for review from the moment it is first <a
+  href='https://www.w3.org/Consortium/Process/#publishing'>published</a>.
+  Working Groups should <a
+  href='https://www.w3.org/Consortium/Process/#formally-addressed'>formally
+  address</a> any substantive review comment about a technical report in a
+  timely manner.</p>
+  <p>Wide review should or must be requested when:</p>
+  <ul>
+  <li>after a <a href="https://www.w3.org/Consortium/Process/#first-wd">First
+  Public Working Draft</a> is published for most documents.
+  <p>Working Groups are often reluctant to make substantive changes to a mature
+  document, so reviewers should get a chance to send substantive technical
+  reviews as early as possible. This is especially important for <a
+  href="#how_to_get_horizontal_review">horizontal reviews</a>.</p>
+  </li>
+  <li>Before a document gets advanced to <a
+  href="https://www.w3.org/Consortium/Process/#transition-cr">Candidate
+  Recommendation</a> or gets updated as a <a
+  href="https://www.w3.org/Consortium/Process/#update-requests">Candidate
+  Recommendation Snapshot</a>.
+  <p>For those, the <a href="https://www.w3.org/Consortium/Process/">W3C
+  Process</a> requires a Group to show that the specification has received wide
+  review.</p>
+  </li>
+  <li>When a document is both reasonably stable and still flexible enough to
+  allow changes where issues are identified</li>
+  <li>When new features are added after a document has already gotten wide
+  review (perhaps requesting a review limited to the new features)</li>
+  </ul>
+  </section>
 
 
 
 <section id="who_to_ask_for_review">
```

Comment:
Though there are some useful additions, a couple of things we put in on purpose were dropped.  I'd also like to reorganise the text slightly, so that the list front-ends the target audiences better and makes it easier to see as a list.  I'll make a suggestion shortly.

Reply:
feel free to make the changes in my branch

Reply:
@plehegar I'm in a bit of a rush, so i'll just paste the source code for my proposed revision of the "Who to ask..." section below.  This is for the whole section. hth

```
<section id="who_to_ask_for_review">
<h2>Who to ask for wide review?</h2>

<p>The objective is to ensure that the entire set of stakeholders of the Web community, including the general public, have  adequate notice of the progress of the Working Group and are able to actually perform reviews of and provide comments on the specification. When considering <a href="https://www.w3.org/Guide/transitions">requests to advance the maturity level of the document</a>, the Director will consider who has been explicitly offered a reasonable opportunity to review the document.</p>

<p>You must publish an updated technical report, with the Status of the Document indicating clearly that you're looking for <em>wide review</em>, before inviting review from other stakeholders.<strong> </strong>Our <a href='https://github.com/w3c/transition-notifier/blob/main/notify.js'>tooling</a> picks up on publications and propagates notifications to <a href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a> automatically (for example, <a href='https://lists.w3.org/Archives/Public/public-review-announce/2021Jun/0008.html'>Candidate Recommendation Snapshot: Payment Request API (Call for Wide Review)</a>).</p>
<p>You should then reach out to:</p>
<ul>
<li>the groups listed in the WG's charter, especially those who manage dependencies.</li>
<li>the groups jointly responsible for a particular document (if any).</li>
<li>the <a href="#how_to_get_horizontal_review">horizontal groups</a>.  If it appears that one of those is not relevant (even if not listed in your charter), talk to them informally to confirm that. <em>Note that sending an email to the public-review-announce list alone is not sufficient to trigger horizontal reviews. You will still need to formally request review from the Horizontal Groups, as described below</em></li>
<li>the general public,  including Web developers, technology providers and implementers, application developers, etc. Consider:
    <ul>
      <li>sending a dedicated announcement to <a href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a> if needed (in case the default notice sent to that list is not enough).</li>
      <li>using blog posts, social media, or other ways of alerting the public to your document and requesting feedback.</li>
      </ul>
      </li>
<li>other groups, at your discretion, even if not in the WG charter, such as other W3C groups, external organizations and SSOs  working on related areas, etc.</li>
</ul>

<p><strong>Tip:</strong> consider tracking your wide review progress using a GitHub issue, such as <a href="https://github.com/w3c/sensors/issues/299">issue #299 of the Sensors API</a>. You can then simply point the Director to the issue.</p>


<p>The reviews provided by the <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal
groups</a>, a subset of a full wide review, do receive special attention and much of the rest of this document focuses on how and when to conduct horizontal reviews.</p>
</section>

```

Reply:
Integrated in https://github.com/w3c/documentreview/pull/21/commits/c00eb79895d88ad4d4c9f90d3ee4a7e97e17c362